app/main.py

```diff
@@ -8,19 +8,46 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people: list) -> list:
-    pass
-    person_list = []
+    person_list = [
+        Person(person_dict["name"],
+               person_dict["age"])
+        for person_dict in people]
+
```

Comment:
We need to improve the code style and naming.
You should not specify what type of variable it is in the variable name.

 That's a good option:
persons = [
        Person(person_data["name"], person_data["age"])
        for persond_data in people
        ]